server/statistics/store.go

```diff
@@ -195,10 +201,14 @@ func (r *RollingStoreStats) GetLoad(k StoreStatKind) float64 {
 		return r.timeMedians[StoreReadBytes].Get()
 	case StoreReadKeys:
 		return r.timeMedians[StoreReadKeys].Get()
+	case StoreReadQuery:
```

Comment:
can we just use r.timeMedians[xxx].Get()

Reply:
done

Reply:
some are `movingAvgs` and others are `timeMedians`

Reply:
I see, updated